app/Main.hs

```diff
@@ -4,19 +4,19 @@ module Main
   ( main
   ) where
 
-import           Actions
```

Comment:
tbh I am not sure why this change appeared because I use hfmt as a plugin for VS Code and it formatted the code in stylish-haskell style

Reply:
Interesting. I do have a config file `.stylish-haskell.yaml` that hfmt uses when I invoke it from the command line at least, maybe the working directory is not set to the project directory when hfmt is invoked from the plugin? That shouldn't be necessary once #22 is fixed, but sorry that's the way it is right now.